sdk/storage/Azure.Storage.Blobs.ChangeFeed/src/LazyLoadingBlobStream.cs

```diff
@@ -171,6 +171,7 @@ private static void ValidateReadParameters(byte[] buffer, int offset, int count)
                 throw new ArgumentNullException($"{nameof(buffer)}", $"{nameof(buffer)} cannot be null.");
             }
 
+#pragma warning disable CA2208 // Instantiate argument exceptions correctly
```

Comment:
same here (see comment above).

Reply:
This one I considered just fixing, however it would potentially be a breaking change as it change the text produced by the exception. If you have no concerns, I'm happy to resolve them (exceptions become:

```c#
new ArgumentOutOfRangeException(nameof(offset), $"{nameof(offset)} cannot be less than 0.");
```

Reply:
We change these messages from time to time. Please apply the resolution.